File: DocumentProject.WebClient/src/app/_models/member.ts
import { BaseModel } from "./abstract/base-model";
import { Organization } from "./organization";

export class Member extends BaseModel {
  email: string | undefined;
  firstName: string | undefined;
  lastName: string | undefined;
  address: string | undefined;
  position: string | undefined;
  phoneNumber: string | undefined;
  photoBase64: string | undefined;
  organizationId: string | undefined;
  organization: Organization | undefined;
  password: string | undefined;
}

File: DocumentProject.WebAPI/DocumentProject.WebAPI/Data/Member.cs
using DocumentProject.WebAPI.Data.Abstract;
using Microsoft.AspNetCore.Identity;

namespace DocumentProject.WebAPI.Data
{
    public class Member: Entity
    {
        public IdentityUser IdentityUser { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public string Address { get; set; }

        public Guid OrganizationId { get; set; }
        public Organization Organization { get; set; }
        public List<Application> Applications { get; set; }
    }
}
